### PR TITLE
Feat/app fe map mc

### PR DESCRIPTION
**Pull request by mcblueglade**
Added Functionality to show map of collectable items and delete an item

### CODE REVIEW

File: src/actions/item.js [hunk 1]
```diff
@@ -108,10 +110,21 @@ export const getMyItems = () => async dispatch => {
...
-export const deleteItem = _id => ({
-  type: types.DELETEITEM,
-  _id
-});
+export const deleteItem = _id => async dispatch => {
+  try {
+    const res = await api.delete(`/items/${_id}`);
+
+    dispatch({
+      type: types.DELETEITEM,
```
Comment (abinormal) [inline src/actions/item.js:118]: Should this be DELETE_ITEM?

File: src/components/Items/MyItemListings.jsx [hunk 2]
```diff
@@ -15,30 +15,30 @@ import PropTypes from 'prop-types';
 export default function MyItemListings(props) {
   const { _id, type, myitems } = props;
   const classes = makeStyles(styles);
-  const paginationColSize = type === 'provide' ? 6 : 12;
+  const paginationColSize = type === 'provide' ? 6 : 6;
```
Comment (Andystyles30) [inline src/components/Items/MyItemListings.jsx:18]: looks like this will always be `6`, or might this change?

**Review by Andystyles30 — APPROVED** [review]

File: src/components/Items/ItemMapPage.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  //  return mytabs;
+  // };
+
+  // createCollectionTabs();
+
+  return (
+    <div className={classNames(classes.main, classes.mainRaised)}>
+      <div className={classes.container}>
+        <GridContainer justify="center">
+          <GridItem xs={12} sm={12} md={8} lg={8}>
```
Comment (abinormal) [inline src/components/Items/ItemMapPage.jsx:52]: Here you only need to put xs={12}  md={8} not the others as the smallest width (xs) is applied first and it'll only change when it gets to md.

File: src/components/Items/MyItemListings.jsx (hunk 2 above)
Comment (abinormal) [inline src/components/Items/MyItemListings.jsx:18]: This is the same number - is this line required?

**Review by abinormal — APPROVED** [review]

**Review by cavalost — APPROVED** [review]

**Pull request merged**